enterprise_gateway/_version.py

```diff
@@ -3,4 +3,4 @@
 # Copyright (c) Jupyter Development Team.
 # Distributed under the terms of the Modified BSD License.
 
-__version__ = '2.1.1.dev2'
+__version__ = '2.1.1.dev4'
```

Comment:
Is this already pushed to art service? I can see 2.1.1.dev4 is already there in art service